extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
Then i don't get it. If the channel never sends any string, use `<event/>` or omit the `<event>` tag. If the channel sends events, specify them in the `<event>` tag.